css-text-4/Overview.bs

```diff
@@ -8337,6 +8337,13 @@ Fullwidth Punctuation Collapsing</h4>
 	the font has proportional ideographs
 	and the fullwidth advance width cannot be reliably determined by measuring glyphs.
 
+	Some fonts have fullwidth punctuation characters
+	whose blank are too small to trim (kern.)
+	UA may not trim (kern)
```

Comment:
```suggestion
	UA may choose not to trim (kern)
```

Otherwise "may not" is ambiguous, and could be seen as meaning the same as "must not"

Reply:
Done, thank you for the review.